src/config.ts

```diff
@@ -1,5 +1,7 @@
 import { Args } from "grimoire-kolmafia";
-import { get } from "libram";
+import { Item } from "kolmafia";
+import { $item, get } from "libram";
+import { stringToWorkshedItem } from "./workshed/utils";
```

Comment:
To prevent circular imports, just define this in `config` instead of `workshed`

Reply:
It really doesn't look like it belongs in config because of how specific it is to the worksheds

Reply:
for what it's worth (not much!), i agree w/ phred, this feels like a config thing despite being very shed-specific.